inst/assets/bibliography.bib

```diff
@@ -1941,235 +1969,217 @@ @article{Haidich2010
+    doi = {10.1073/pnas.0506580102},
     year = {2005}
 }
 
+
 @article{Kou2020,
-    author = {Kou Y},
+    author = {Yan Kou and Xiaomin Xu and Zhengnong Zhu and Lei Dai and Yan Tan},
     title = {Microbe-set enrichment analysis facilitates functional interpretation of microbiome profiling data},
     journal = {Scientific Reports},
     volume = {10},
     number = {1},
     pages = {21466},
-    url = {https://www.nature.com/articles/s41598-020-78511-y},
+    doi = {10.1038/s41598-020-78511-y},
     year = {2020}
 }
 
+
 @article{Langfelder2008,
-    author = {Langfelder},
-    title = {WGCNA: an R package for weighted correlation network analysis},
+    author = {Peter Langfelder and Steve Horvath},
+    title = {{WGCNA}: an R package for weighted correlation network analysis},
     journal = {BMC Bioinformatics},
     volume = {9},
-    number = {},
     pages = {559},
-    url = {https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-9-559},
+    doi = {https://doi.org/10.1186/1471-2105-9-559},
     year = {2008}
 }
 
+
 @article{Geistlinger2023,
-    author = {Geistlinger L},
-    title = {BugSigDB captures patterns of differential abundance across a broad range of host-associated microbial signatures},
+    author = {Ludwig Geistlinger and Chloe Mirzayi and Fatima Zohra and Rimsha Azhar and Shaimaa Elsafoury and Clare Grieve and Jennifer Wokaty and Samuel David Gamboa-Tuz and Pratyay Sengupta and Issac Hecht and Aarthi Ravikrishnan and Rafael S. Gonçalves and Eric Franzosa and Karthik Raman and Vincent Carey and Jennifer B. Dowd and Heidi E. Jones and Sean Davis and Nicola Segata and Curtis Huttenhower and Levi Waldron},
+    title = {{BugSigDB} captures patterns of differential abundance across a broad range of host-associated microbial signatures},
     journal = {Nature Biotechnology},
-    volume = {},
-    number = {},
-    pages = {},
-    url = {https://doi.org/10.1038/s41587-023-01872-y},
+    volume = {42},
+    number = {5},
+    pages = {790-802},
+    doi = {https://doi.org/10.1038/s41587-023-01872-y},
     year = {2023}
 }
-
-@article{Legendre2001,
-  author = {Legendre, Pierre and Gallagher, Eugene D.},
-  title = {Ecologically Meaningful Transformations for Ordination of Species Data},
-  journal = {Oecologia},
-  month = oct,
-  volume = {129},
-  number = {2},
-  pages = {271--280},
-  doi = {10.1007/s004420100716},
-  year = {2001}
-}
-
-@article{Schloss2024rarefaction1,
-    author = {Patrick D. Schloss},
-    title = {Rarefaction is currently the best approach to control for uneven sequencing effort in amplicon sequence analyses},
-    journal = {{mSphere}},
-    volume = {9},
-    number = {2},
-    pages = {e00354-23},
-    doi = {10.1128/msphere.00354-23},
-    year = {2024}
-}
-
-@article{Schloss2024rarefaction2,
-    author = {Patrick D. Schloss},
-    title = {Waste not, want not: revisiting the analysis that called into question the practice of rarefaction},
```

Comment:
Why these are removed?

Reply:
I would keep the refs that are there. We might like to cite them later when finalizing the text.